### PR TITLE
Fix: Added getPadding to timeline to fix padding issue (fixes #136)

### LINKED ISSUE #136: Timeline displays data points(values) that are in the past
**Describe the bug**
When values are added to the timeline which are in the past compared to the upper and lower limit, they are still displayed on the timeline. However, the values are not displayed when they in the future, when compared to the upper and lower limit.

**To Reproduce**
See working example below:
https://codesandbox.io/s/data-points-displaying-outside-range-9rvy4

**Expected behavior**
I would expect the values in the past NOT to be displayed as they are out of range within the upper and lower limit.

**Screenshots**
Screen Shot 2019-12-27 at 13 51 25

**Desktop (please complete the following information):**
-   OS: Mac
-   Browser: Chrome

### PR DESCRIPTION
**Pull request by saket2403**
**What is the purpose of this pull request? (Check any that's applicable)**

-   [x] Feature
-   [ ] Bug fix
-   [ ] Dependency updates
-   [ ] Documentation updates
-   [ ] Build related changes
-   [ ] Changes an existing functionality
-   [ ] Other, please explain:

**Does this introduce a breaking change?**

-   [ ] Yes
-   [x] No

**What changes did you make? (Give an overview)**
Added a getPadding function to apply custom padding to timeline if provided by user else use default values.

**Add any screenshots**
image

### CODE REVIEW

**Comment by abhijit945** [comment]
I would amend the commit to what was actually added, since there is already a lot of `fix padding issue` type PRs.

**Comment by saket2403** [comment]
> Can you add few unit tests for this?

Added in 966433e

File: src/test/unit/controls/Timeline/Timeline-spec.js [hunk 1]
```diff
@@ -217,6 +217,30 @@ describe("Timeline", () => {
...
+            input.content = null;
+            data.tasks = [];
+            const cfg = timeline.config;
+            expect(input.bindTo).toBe("");
+            expect(input.axis).toEqual({});
+            expect(data.tasks).toEqual([]);
+            expect(cfg.bindTo).toBe("#testCarbonTimeline");
+            expect(cfg.axis.x.lowerLimit).toBe(axisJSON.x.lowerLimit);
+            expect(cfg.axis.x.upperLimit).toBe(axisJSON.x.upperLimit);
+            expect(cfg.padding.hasCustomPadding).toBe(true);
```
Comment (abhijit945) [inline src/test/unit/controls/Timeline/Timeline-spec.js]: The test doesnt check for the padding property. Neither does it check if the content width and height properties have changed.

Comment (saket2403) [reply src/test/unit/controls/Timeline/Timeline-spec.js]: Updated in 763c968

**Pull request merged**